src/plugins/share/public/components/tabs/embed/embed_content.tsx

```diff
@@ -67,10 +65,6 @@ export const EmbedContent = ({
   const [anonymousAccessParameters] = useState<AnonymousAccessState['accessURLParameters']>(null);
   const [usePublicUrl] = useState<boolean>(false);
 
-  useEffect(() => {
-    if (objectType !== 'dashboard') setIsNotSaved();
-  }, [url, setIsNotSaved, objectType]);
-
```

Comment:
We remove this function because the resulting value from the computation is not used.